src/components/Export.tsx

```diff
@@ -379,9 +396,9 @@ export default function Export(inputProps: ExportButtonProps): ReactElement {
     // (475 MB predicted vs. 70 MB actual)
     // TODO: Is there a way to concretely determine this?
     const compressionRatioBitsPerPixel = 3.5; // Actual value 2.7-3.0, overshooting to be safe
-    const canvasExportDimensions = props.getCanvasExportDimensions();
-    const maxVideoBitsResolution =
-      canvasExportDimensions[0] * canvasExportDimensions[1] * totalFrames * compressionRatioBitsPerPixel;
+    // Use current canvas dimensions as an approximation (instead of the export dimensions)
+    const canvas = props.getCanvas();
+    const maxVideoBitsResolution = canvas.width * canvas.height * totalFrames * compressionRatioBitsPerPixel;
 
```

Comment:
This is a performance optimization, since currently `getCanvasExportDimensions()` basically does a re-render under the hood to measure the header + footer.